pkg/core/resources/model/resource_test.go

```diff
@@ -259,8 +259,22 @@ var _ = Describe("ComputePolicyRole", func() {
 					IdleTimeout: &kube_meta.Duration{Duration: 123 * time.Second},
 				}).
 				Build().Spec,
-			namespace:    "kuma-demo",
-			expectedRole: mesh_proto.WorkloadOwnerPolicyRole,
+			namespace:   core_model.NewNamespace("kuma-demo", false),
+			expectedErr: "it's not allowed to mix 'to' and 'from' arrays in the same policy",
+		}),
+		Entry("system policy with both from and to", testCase{
+			policy: builders.MeshTimeout().
+				WithMesh("mesh-1").WithName("name-1").
+				WithTargetRef(builders.TargetRefMesh()).
+				AddTo(builders.TargetRefMesh(), meshtimeout_api.Conf{
+					IdleTimeout: &kube_meta.Duration{Duration: 123 * time.Second},
+				}).
+				AddFrom(builders.TargetRefMesh(), meshtimeout_api.Conf{
+					IdleTimeout: &kube_meta.Duration{Duration: 123 * time.Second},
+				}).
+				Build().Spec,
+			namespace:    core_model.NewNamespace("kuma-system", true),
+			expectedRole: mesh_proto.SystemPolicyRole,
 		}),
```

Comment:
So consumer policies can't mix both? Are we sure this isn't an issue? I'd love to see more tests with legacy stuff to make sure we're not breaking backwards compat.

For .e.g I'd expect the following to work:

```yaml
kind: MeshTimeout
namespace: kuma-system
spec:
  to:
   - targetRef:
        kind: MeshService
        name: old_ns_svc_2030
  from:
   - targetRef:
        kind: MeshService
        name: old_ns_svc_2030
```

Sure from reading the code it looks like it works. But we all know we write bugs so a test would be nice!

Reply:
The case you mentioned was already there:
```go
Entry("system policy with both from and to", testCase{
	policy: builders.MeshTimeout().
		WithMesh("mesh-1").WithName("name-1").
		WithTargetRef(builders.TargetRefMesh()).
		AddTo(builders.TargetRefMesh(), meshtimeout_api.Conf{
			IdleTimeout: &kube_meta.Duration{Duration: 123 * time.Second},
		}).
		AddFrom(builders.TargetRefMesh(), meshtimeout_api.Conf{
			IdleTimeout: &kube_meta.Duration{Duration: 123 * time.Second},
		}).
		Build().Spec,
	namespace:    core_model.NewNamespace("kuma-system", true),
	expectedRole: mesh_proto.SystemPolicyRole,
}),
```
added 2 other cases that were not covered